extension/_locales/en/messages.json

```diff
@@ -0,0 +1,82 @@
+{
+    "extensionName": {
+      "message": "Firefox Translations",
+      "description": "Name of the extension"
+    },
+    "extensionDescription": {
+      "message": "Translate websites in your browser without using the cloud.",
+      "description": "Description of the extension."
+    },
+    "loadingTranslationEngine": {
+      "message": "Loading Translation Engine",
+      "description": "Tells the user the translation engine is being loaded."
+    },
+    "translationLoadedWithErrors": {
+      "message": "Automatic translation is enabled but we found errors.",
+      "description": "Tells the user the translation engine was loaded but errors were found."
+    },
+    "checksumErrorsDownloadingEngine": {
+      "message": "We found checksum errors when downloading the translation engine.",
+      "description": "Tells the user there were errors downloading the translation engine."
+    },
+    "notfoundErrorsDownloadingEngine": {
+      "message": "We could not find the translation engine to download.",
+      "description": "Tells the user the translation engine could not be found to download."
+    },
+    "nodataDownloadingEngine": {
+      "message": "We found no data when downloading the translation engine.",
+      "description": "Tells the user the translation engine had not data when downloading."
+    },
+    "translationEnabled": {
+      "message": "Automatic Translation enabled.",
+      "description": "Tells the user the automatic translation was enabled."
+    },
+    "translationEnabledNoOT": {
+      "message": "Automatic Translation enabled but translation of forms is not supported for this language.",
+      "description": "Tells the user the automatic translation was enabled but form translations is not supported."
+    },
+    "timeoutDownloadingEngine": {
+      "message": "Timeout downloading the translation engine.",
+      "description": "Tells the user the download of the translation engine timed out."
+    },
+    "errorLoadingWasm": {
+      "message": "We found an error loading the translation engine.",
+      "description": "Tells the user we found and error loading the translation engine."
+    },
+    "statsMessage": {
+      "message": "Translation enabled. Words translated-per-second: $WORDSPERSECOND$.",
+      "description": "Tells the user that translation is enabled and the current performance metric of words translated per second.",
+      "placeholders": {
+        "wordspersecond" : {
+          "content" : "$1",
+          "example" : "500"
+        }
+      }
+    },
+    "downloadProgress": {
+      "message": "Downloaded $RECEIVEDLENGTH$ of $CONTENTLENGTH$.",
```

Comment:
What is the unit of measure here? "100 of 1000" what?

Reply:
The unit is bytes. We have in all 3-4 items to download, with a maximum of 16-20 MB per item. Is there something that we should add here?

Reply:
I'm looking at [the code](https://github.com/mozilla/firefox-translations/blob/33733e95896c946d239eeeeab5063052dc4f0e7d/extension/controller/translation/translationWorker.js#L445) but can't seem to find where you are formatting the variables. Is it just the number at the moment, without unit? Probably something like `Downloaded (bytes): $RECEIVEDLENGTH$ of $CONTENTLENGTH$.` would work, and you'll avoid plurals (which are not supported in this format).

Reply:
Examples:

https://github.com/mozilla/firefox-translations/blob/33733e95896c946d239eeeeab5063052dc4f0e7d/extension/model/engineRegistry.js#L9

https://github.com/mozilla/firefox-translations/blob/33733e95896c946d239eeeeab5063052dc4f0e7d/extension/model/modelRegistry.js#L11

So, these are just numbers. I am sorry if the meaning of the word "unit" means something specific in extension context as I took a general meaning for this word.

Reply:
Unit = unit of measurement.

I wanted to make sure we weren't hardcoding something together with the number, e.g. 1000 would magically become "1000 bytes" (hardcoding spaces and unit of measurement).

Reply:
Great. We are not hardcoding the unit with the actual number 👍🏾